.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-  - "2.7"
+  - "3.6"
```

Comment:
I'm not familiar with this repo but it may be good to run tests against more recent versions of Python too? (though it gets tricky with the docker_push.sh script and stuff like that, so probably something to address in another PR).

Reply:
Picked this since it matches what we use internally - agreed that another PR can look into extending the test matrix.

Reply:
there's no tests 😓